packages/world/src/systems/CraftSystem.sol

```diff
@@ -70,33 +133,42 @@ contract CraftSystem is System {
         currentInput++;
       }
 
-      // Handle special case for coal ore
-      if (recipeType == ObjectTypes.CoalOre) {
+      // Handle special case for ores
+      if (recipeType.isOre()) {
```

Comment:
Do we need this now? Ores are not used in any other recipes

And if they are eg to craft some dense ore block we dont want to burn them, as we'll just make that recipe reversible

We dont have any at the moment though so should just remove it imo

Reply:
ah good point, yeah we should remove